packages/codegen/generate.gql.js

```diff
@@ -245,4 +245,11 @@ module.exports = {
     generateGqlQueries,
     generateQueryWhereInput,
     generateQuerySortBy,
+    genGetAllGQL,
```

Comment:
Should it be consistent with other names? (`gen` vs `generate`)

Reply:
Gen and Generate prefix has differences in implementation. genGetAllGQL suppose to be close to genGetCountGQL/genCreateGQL/genUpdateGQL and other.

Reply:
Yep, I mean why not to use `generate` on every name? Because now you have half of exported members to have `generate` prefix, and other half is `gen`, so its inconsistent

Reply:
make sense, will change